spec/factories/claim/interim_claims.rb

```diff
@@ -6,6 +6,13 @@
     case_concluded_at nil
   end
 
+  trait :interim_trial_start_fee do
+    after(:build) do |claim|
+      claim.fees << build(:interim_fee, :trial_start, claim: claim)
```

Comment:
If you use this _trait_ while passing a **estimated_trial_length**, e.g.

```ruby
create(:interim_claim, : interim_trial_start_fee, estimated_trial_length: 3)
```

what value does the claim have in the end? 

I would say its 2, but should probably be 3. We should really ensure any input values should take priority over any factory ones.

Reply:
it will be 2, yes. and your right attributes set on the claim explicitly should take precedence.  will take a look at how that can be achieved.